perfkitbenchmarker/data/Configfile_blaze

```diff
@@ -0,0 +1,92 @@
+#==================================================================================================
```

Comment:
Could you document somewhere the changes you made to these config files and why you made them? Put it where ever you think is appropriate - not necessarily in the files themselves